layouts/partials/openapi/render-object-table.html

```diff
@@ -155,12 +155,10 @@
       adhering to a regex pattern
 
     * `anchor`: optional HTML element id for the target type, which will be used to link to it.
-
-    The title has a higher priority than anything else.
 */}}
 {{ define "partials/object-type-or-title" }}
     {{ $type := "object" }}
-    {{ if .title }}
+    {{ if and .title (or .properties (not (or .additionalProperties .patternProperties))) }}
```

Comment:
could you add a comment explaining what this logic does in words, and why?

Reply:
Oops, yes. I wanted to an forgot. Docs added.